ooniprobe/Model/NetworkMeasurement.mm

```diff
@@ -441,19 +230,10 @@ -(void)run{
 }
 
 -(void) run_test {
-    [super run_test];
     NSBundle *bundle = [NSBundle mainBundle];
     NSString *path = [bundle pathForResource:@"global" ofType:@"txt"];
```

Comment:
Are these two variables used?

Reply:
yes, they are removed in the other branch